### PR TITLE
[CMSP-150] add release note for mu-plugin 1.4.4 and WPCM update

### PR DESCRIPTION
**Pull request by jazzsequence**
## Summary

**[Release Notes](https://docs.pantheon.io/release-notes)** - Adds release note for changes made in https://github.com/pantheon-systems/wordpress-composer-managed/pull/125, https://github.com/pantheon-systems/wordpress-composer-managed/pull/127 https://github.com/pantheon-systems/pantheon-mu-plugin/pull/49

### Dependencies and Timing

**Release**:
- [x] When ready
- [ ] After date: $DATE

--------------------------------------------------

## Post Launch

**Do not remove** - To be completed by the docs team upon merge:

- [ ] Redirect `/old-path/` => `/new-path/` (if applicable)
- [ ] Include/exclude pages ^ respectively within docs search service provider (if applicable)

### CODE REVIEW

File: source/releasenotes/2024-06-04-pantheon-mu-plugin-1-4-4-wordpress-composer-managed-update.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+title: Pantheon MU Plugin v1.4.4 and WordPress (Composer Managed) update
+published_date: "2024-06-04"
+categories: [wordpress, plugins]
+---
+
+The latest [1.4.4 update](https://github.com/pantheon-systems/pantheon-mu-plugin/releases) of the Pantheon MU Plugin is now available. This update fixes the WordPress Multisite network setup process by by ensuring the correct instructions for WordPress Multisite setup on Pantheon environments are always shown. This update will be included with the next WordPress release. Composer-based WordPress installs can get the update today by running `composer update`.
+
+The latest WordPress (Composer Managed) release updates the configuration file filename displayed in the network setup screen to specify `config/application.php` to apply your configuration changes to, rather than the default `wp-config.php` for non-Bedrock Composer installs. This update will be pushed out to all WordPress (Composer Managed)-based installs alongside the MU Plugin update.
```
Comment (scottbuscemi) [inline source/releasenotes/2024-06-04-pantheon-mu-plugin-1-4-4-wordpress-composer-managed-update.md:9]: @jazzsequence Please link to the WPCM repo and mention it's still in EA :)

**Review by scottbuscemi — DISMISSED** [review]

**Comment by IngridKwok** [comment]
Looks good. @stevector @rachelwhitton

File: source/releasenotes/2024-06-04-pantheon-mu-plugin-1-4-4-wordpress-composer-managed-update.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+---
+title: Pantheon MU Plugin v1.4.4 and WordPress (Composer Managed) update
+published_date: "2024-06-04"
+categories: [wordpress, plugins]
+---
+
+The latest [1.4.4 update](https://github.com/pantheon-systems/pantheon-mu-plugin/releases) of the Pantheon MU Plugin is now available. This update fixes the WordPress Multisite network setup process by by ensuring the correct instructions for WordPress Multisite setup on Pantheon environments are always shown. This update will be included with the next WordPress release. Composer-based WordPress installs can get the update today by running `composer update`.
+
+The latest [WordPress (Composer Managed) (Early Access)](https://github.com/pantheon-upstreams/wordpress-composer-managed) release updates the configuration file filename displayed in the network setup screen to specify `config/application.php` to apply your configuration changes to, rather than the default `wp-config.php` for non-Bedrock Composer installs. This update will be pushed out to all WordPress (Composer Managed)-based installs alongside the MU Plugin update.
```
Comment (stevector) [inline source/releasenotes/2024-06-04-pantheon-mu-plugin-1-4-4-wordpress-composer-managed-update.md:9]: @jazzsequence what's the relationship between these two changes? Does the WordPress Composer Manage change also relate to Multisite?

If these are actually to unrelated changes that are just happening on the same day, I think we should do two separate release notes.

File: source/releasenotes/2024-06-04-pantheon-mu-plugin-1-4-4-wordpress-composer-managed-update.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+
+The WordPress (Composer Managed) update also includes a Composer script to automatically update the Composer `platform.php` value to match the version of PHP in your `pantheon.yml`. **Note:** [Roots Bedrock](https://roots.io/bedrock/docs/installation/#requirements) does not support below PHP 8.0.
```
Comment (stevector) [inline source/releasenotes/2024-06-04-pantheon-mu-plugin-1-4-4-wordpress-composer-managed-update.md:11]: > The WordPress (Composer Managed) update

I don't think we should do anything different right now, but that phrasing makes me think we should figure out a way to apply version numbers to "WordPress (Composer Managed)". With the vast majority of upstream updates we get to refer to the version numbers of WordPress and Drupal core. Here it's just a repo with new commits in main.

**Review by stevector — CHANGES_REQUESTED** [review]
I think we should make the relationship between these releases clearer. Or if there is not a relationship between them, make two separate notes.

File: source/releasenotes/2024-06-04-pantheon-mu-plugin-1-4-4-wordpress-composer-managed-update.md (hunk 2 above)
Comment (jazzsequence) [reply source/releasenotes/2024-06-04-pantheon-mu-plugin-1-4-4-wordpress-composer-managed-update.md:9]: Yes, they relate to each other in that the page receiving the updates is _in the mu-plugin_ but the instructions need to be differentiated _in the Composer Managed_ upstream. I can make that clearer.

File: source/releasenotes/2024-06-04-pantheon-mu-plugin-1-4-4-wordpress-composer-managed-update.md (hunk 3 above)
Comment (jazzsequence) [reply source/releasenotes/2024-06-04-pantheon-mu-plugin-1-4-4-wordpress-composer-managed-update.md:11]: I could give it a date-based update, e.g.
```suggestion
The WordPress (Composer Managed) `2024-06-04` update also includes a Composer script to automatically update the Composer `platform.php` value to match the version of PHP in your `pantheon.yml`. **Note:** [Roots Bedrock](https://roots.io/bedrock/docs/installation/#requirements) does not support below PHP 8.0.
```

This is how it's tracked in [the changelog](https://github.com/pantheon-systems/wordpress-composer-managed/blob/default/CHANGELOG.md#2024-06-04)

**Comment by jazzsequence** [comment]
@stevector I've restructured the update so it makes a bit more sense narratively and connects the two things a bit better (hopefully).

File: source/releasenotes/2024-06-04-pantheon-mu-plugin-1-4-4-wordpress-composer-managed-update.md (hunk 3 above)
Comment (jazzsequence) [reply source/releasenotes/2024-06-04-pantheon-mu-plugin-1-4-4-wordpress-composer-managed-update.md]: Updated the note to link to the changelog in the Composer Managed upstream.

File: source/releasenotes/2024-06-05-wordpress-6-5-4.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+title: WordPress 6.5.4 Maintenance Update
+published_date: "2024-06-05"
+categories: [wordpress, action-required]
+---
+
+[WordPress 6.5.4](https://wordpress.org/news/2024/06/wordpress-6-5-4-maintenance-release/) is now available on the platform.
+
+<h3>Highlights</h3>
+
+* [5 bug fixes in Core](https://core.trac.wordpress.org/query?status=closed&milestone=6.5.4&group=status&col=id&col=summary&col=owner&col=type&col=priority&col=component&col=version&col=keywords&order=priority)
```
[the remark this answers is not in the document]
Comment (pwtyler) [reply source/releasenotes/2024-06-05-wordpress-6-5-4.md]: They are related because 6.5.4 ships this and every other recent mu update in the upstream.

**Review by stevector — APPROVED** [review]
Thanks @jazzsequence!

**Comment by stevector** [comment]
I'm dismissing the "1 requested change" from @scottbuscemi because I can see the changes have been made.
